pkgs/development/python-modules/dukpy/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, setuptools
+, mutf8
+}:
+
+buildPythonPackage rec {
+  pname = "dukpy";
+  version = "0.4.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Z37HEC0cHFEffvkYB46AmXeNvOp8rz1qKipy9yqi0TU=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  propagatedBuildInputs = [ mutf8 ];
+
+  pythonImportsCheck = [ "dukpy" ];
+
+  meta = {
+    description = "Simple JavaScript interpreter for Python";
+    homepage = "https://github.com/amol-/dukpy";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ ruby0b ];
```

```suggestion
    maintainers = with lib.maintainers; [ ruby0b ];
    mainProgram = "dukpy";
```
https://github.com/amol-/dukpy/blob/4963346467bc32f07f7cc64d43bda7eb48c1c437/setup.py#L77-L82